tests/event-log/event-log-level.spec.ts

```diff
@@ -0,0 +1,130 @@
+import chaiAsPromised from 'chai-as-promised';
+import { computeCid } from '../../src/utils/cid.js';
+import { EventLogLevel } from '../../src/event-log/event-log-level.js';
+import { TestDataGenerator } from '../utils/test-data-generator.js';
+import chai, { expect } from 'chai';
+
+chai.use(chaiAsPromised);
+
+let eventLog: EventLogLevel;
+
+describe('EventLogLevel Tests', () => {
+  describe('append tests', () => {
+    before(async () => {
+      eventLog = new EventLogLevel({ location: 'TEST-EVENTLOG' });
+      await eventLog.open();
+    });
+
+    beforeEach(async () => {
+      await eventLog.clear();
+    });
+
+    after(async () => {
+      await eventLog.close();
+    });
+
+    it('appends a tenant namespaced entry into leveldb', async () => {
+      const { requester, message } = await TestDataGenerator.generateRecordsWrite();
+      const messageCid = await computeCid(message);
+
+      const watermark = await eventLog.append(requester.did, messageCid.toString());
+
+      for await (const [key, value] of eventLog.db.iterator()) {
+        expect(key).to.include(watermark);
+        expect(value).to.equal(messageCid.toString());
+      }
+    });
+
+    it('maintains order in which events were appended', async () => {
+      const { requester, message } = await TestDataGenerator.generateRecordsWrite();
+      const messageCid = await computeCid(message);
+
+      await eventLog.append(requester.did, messageCid.toString());
```

Comment:
All occurrences of `messageCid.toString()` can just be `messageCid` in this file. (It probably threw you off because it used to be a CID object)